(docs): integration tests add chalk check (#244)
chalk-ai/docs: integration-tests.mdx

---

> In the previous section, we talked about how to [unit-testindividual resolvers](/docs/unit-tests)—this process is easy since each resolver is a callable Python functions and you can assert on the expected output. In this section we will talk about writing integration tests. Chalk provides two features that make integration testing really easy:

Author: ```suggestion
In the previous section, we talked about how to [unit-test individual resolvers](/docs/unit-tests)—this process is easy since each resolver is a callable Python functions and you can assert on the expected output. In this section we will talk about writing integration tests. Chalk provides two features that make integration testing really easy:
```

> - Branch deployment,

Author: ```suggestion
- branch deployments,
```

> - The `check` method of the python Chalk client.

Author: ```suggestion
- the `check` method of the python Chalk client.
```

> With branch deploys, you can test your changes in an isolated environment before

Author: ```suggestion
With branch deployments, you can test your changes in an isolated environment before
```

> With branch deploys, you can deploy and test your changes in an isolated environment before

Author: ```suggestion
With branch deploys, you can test your changes in an isolated environment before
```
